hutch_python/tests/test_log_setup.py

```diff
@@ -84,16 +85,20 @@ def clear(queue):
 
 def assert_is_info(queue):
     clear(queue)
-    logger.info('hello')
-    logger.debug('goodbye')
-    assert 'hello' in queue.get(block=False).getMessage()
-    assert queue.empty()
+    info_msg = str(uuid.uuid4())
+    debug_msg = str(uuid.uuid4())
```

Comment:
This was changed to avoid logging cross contamination?

Reply:
I thought it would be better to use random garbles of strings rather than messages that could theoretically be matched now that I'm searching through potentially several messages instead of exactly one.